web/gui-v2/src/components/ListViewTable.jsx

```diff
@@ -242,10 +272,38 @@ const ListViewTable = ({
       continent: useQueryParamString('continent', initialVal('continent')),
       stage: useQueryParamString('stage', initialVal('stage')),
       // ...
+
+      all_pubs: useQueryParamString('all_pubs', initialVal('all_pubs')),
```

Comment:
again I wonder if iteration could make sense here, it's a lot of boilerplate

Reply:
Well, looks like there's a problem after all - refreshing the page leads to the filter state bouncing back and forth:

https://github.com/georgetown-cset/parat/assets/22353962/4692be32-2cb8-4a59-8d7d-d2d3b424dca2

Reply:
Yikes, good catch. Is this a pre-existing issue? If so maybe makes sense to open a separate issue for it

Reply:
Tracked in #131